src/serialization/sb3.js

```diff
@@ -416,13 +422,15 @@ const serializeComments = function (comments) {
  */
 const serializeTarget = function (target) {
     const obj = Object.create(null);
+    const extensionIDs = new Set();
```

Comment:
I think I would rather keep the pattern of the rest of the file of each function initializing and returning a sub structure. E.g. I think serializeBlocks should be the one to initialize extensionIDs (and then return it) since it's the function that's populating it.